interview-questions.md

```diff
@@ -7,52 +7,70 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What are props in React?
 
-  Your answer:
+  Your answer: Props in react is a keyword. Props is used to pass information from one arguement to the next. 
 
   Researched answer:
 
-
+  Props short for properties is a keyword in React for passing information from one component to another. Props are only passed in one direction, from parent to child and cannot be updated, they are "read only.
 
 2. What is a DOM event?
 
-  Your answer:
+  Your answer: DOM events are the movements and actions form the mouse and keyboard.
 
   Researched answer:
 
+  DOM events are (MouseEvent), (TouchEvent) and (KeyboardEvent). The DOM is always listening for changes. As the user moves their mouse across the screen and or touch their keyboard, the DOM reacts by changing the view to represent the cursor's position.
 
 
 3. What is object-oriented programming? How is it different than functional programming?
 
-  Your answer:
+  Your answer: OPP is computer language that is odject structured like Ruby. Its different from functional because OOP is written for human understading.
 
   Researched answer:
 
+  Object Oriented Programming (OOP) is a computer programming model that organizes software design around data, or objects unlike, functional programming which require logic.
 
 
 4. What is the difference between a Float and an Integer in Ruby?
 
-  Your answer:
+  Your answer: 
+  
+  The difference between a Float and an interger are, intergers are whole numbers and Float are decimal numbers. 
 
   Researched answer:
 
-
+  The difference between a Float and a Integer, a float in a Ruby program is a number that contains a decimal point. Ruby will consider any number written without decimals as an integer and any number written with decimals as a float. 
 
 5. Ruby has an implicit return. What does that mean?
 
   Your answer:
 
-  Researched answer:
+  An implicit return in Ruby means the same as undefine in javascript 
 
+  Researched answer:
 
+  Implicit returns means that if a return is the last expression in a path of execution, there's no need for the return keyword.
```

Comment:
Make sure that all of your researched answers are formed _**in your own words**_!